src/core/k8.mjs

```diff
@@ -661,19 +661,45 @@ export class K8 {
    * @param podName pod name
    * @param localPort local port
    * @param podPort port of the pod
+   * @param namespace namespace of the pod (optional)
    */
-  async portForward (podName, localPort, podPort) {
-    const ns = this._getNamespace()
+  async portForward (podName, localPort, podPort, namespace = null) {
+    const ns = namespace || this._getNamespace()
```

Comment:
same comment on passing the namespace as above.

Reply:
same response